extension/src/test/suite/experiments/workspace.test.ts

```diff
@@ -152,8 +152,8 @@ suite('Workspace Experiments Test Suite', () => {
 
       await testFile('params.yaml')
       await testFile('dvc.yaml')
-    })
-  }).timeout(WEBVIEW_TEST_TIMEOUT)
+    }).timeout(WEBVIEW_TEST_TIMEOUT)
```

Comment:
There seems to be an issue with tests inside of describe blocks not getting the timeout. This test was failing with a timeout of `6000` until I moved the `.timeout`. This PR is also failing with:

```
  1) Experiments Test Suite
       Sorting
         should not show duplicate rows when sorted:
     Error: Timeout of 6000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (d:\a\vscode-dvc\vscode-dvc\extension\dist\test\suite\experiments\index.test.js)
  	at listOnTimeout (node:internal/timers:569:17)
  	at process.processTimers (node:internal/timers:512:7)
```

This sorting test is also relying on its parent describe block for its set webview timeout.

Reply:
Reviewed `mocha` prs and issues, but not seeing anything. It's also strange that we're hitting all the timeouts today when we haven't merged anything recently related to `mocha` (as far as I can tell).

Reply:
I would guess there shouldn't be multiple `describe` blocks nested (see below)

Reply:
Why shouldn't there be multiple `describe` blocks in a block? The mocha docs show examples of this. Or am I misunderstanding something?

Reply:
> Why shouldn't there be multiple describe blocks in a block? The mocha docs show examples of this. Or am I misunderstanding something?

Have a look at [this](https://stackoverflow.com/questions/55415555/nested-describe-and-behaviour-of-dynamically-created-its) and [this](https://mochajs.org/#run-cycle-overview). For our purposes, we want to avoid extra complexity/parallelization. I have very little faith that `suite -> describe -> describe -> it` acts the same way as `suite -> describe -> it`

> This sorting test is also relying on its parent describe block for its set webview timeout.

`describe` blocks should not have `.timeout` added to them. If you see it that is an error.

Reply:
> Have a look at [this](https://stackoverflow.com/questions/55415555/nested-describe-and-behaviour-of-dynamically-created-its) and [this](https://mochajs.org/#run-cycle-overview). For our purposes, we want to avoid extra complexity/parallelization. I have very little faith that suite -> describe -> describe -> it acts the same way as suite -> describe -> it

Makes sense, thanks for the clarifications! I'm only seeing one example of `suite -> describe -> describe ->` which is our `Custom Plots Creation`. None of those tests have failed that I can see, but it would still be quick to remove the extra nesting there.

> describe blocks should not have .timeout added to them. If you see it that is an error.

Oh, ok that explains the issue. I actually couldn't find documentation on `.timeout` which explains how I missed this 🤦‍♀️. 

The tests appear to be running normally again but I'll go ahead and fix the issues mentioned above to ensure this doesn't happen again.